locale/en-US/messages.properties

```diff
@@ -1 +1,8 @@
-config.labelFromProperties=Label defined in a properties file
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+config.rule.item.controls.edit.label=✏
```

Comment:
Because I didn't explain you well, you may not have understood what I wanted to say. I think we should use `✎` (LOWER RIGHT PENCIL, U+270E) instead of `✏` (PENCIL, U+270F). Because `✏` looks like hyphen for me.